cookbook/core/remote_flyte/customizing_resources.py

```diff
@@ -26,7 +25,11 @@
 # #. ``mem``
 # #. ``gpu``
 #
-# The ``storage`` resources option is not yet supported, but coming soon
+# It is not necessary to adjust admission controllers for gpu nodes, but they will need to be configured with a taint. 
+#
+# To ensure that the regular tasks do not get shceduled on a gpu node, set up flytepropeller using the following `configuration <https://github.com/flyteorg/flytepropeller/blob/master/config.yaml#L51,L56>`_.
```

Comment:
since master is liable to change, can we use a specific commit sha in the link?

Reply:
Done.